src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -250,6 +252,22 @@ public void testPullPrivateRepoWithoutAuth() throws Exception {
   }
 
   @Test
+  public void testBuildImageIdWithBuildargs() throws Exception {
+    assumeTrue("Docker API should be at least v1.21 to support buildargs while building, got "
```

Comment:
I don't think we need this `assumeTrue`. Docker API has had build params since 1.18 https://docs.docker.com/engine/reference/api/docker_remote_api_v1.18/#build-image-from-a-dockerfile